src/clientSideScene/sceneInfra.ts

```diff
@@ -308,8 +306,17 @@ class SceneInfra {
     this.scene.background = null
 
     // CAMERA
+    const camHeightDistanceRatio = 0.5
+    const baseUnit: BaseUnit =
+      JSON.parse(localStorage?.getItem(SETTINGS_PERSIST_KEY) || ('{}' as any))
+        .baseUnit || 'in'
```

Comment:
```suggestion
        .baseUnit || 'mm'
```

i made everything else default mm